src/connectors/impls/gbq/writer.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021, The Tremor Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod sink;
+
+use crate::connectors::impls::gbq::writer::sink::GbqSink;
+use crate::connectors::prelude::*;
+use crate::connectors::{Connector, ConnectorBuilder, ConnectorConfig, ConnectorType};
+use serde::Deserialize;
+use tremor_pipeline::ConfigImpl;
+
+#[derive(Deserialize, Clone)]
+pub(crate) struct Config {
+    pub table_id: String,
+    pub connect_timeout: u64,
```

Comment:
we have all timeouts in all the other connectors (and everywhere else) in nanoseconds. Would you mind changing these to nanos as well? I know seconds might be a little more applicable for such timeouts, but for the sake of consistency etc.

Reply:
Good catch!

Reply:
Good point, seconds might be too much, I guess people might want to set timeouts which are in tens of ms. I hope some day we'll have some nice syntax for passing timeouts, because manually counting zeroes isn't fun :).
Anyway, it's done, switched to nanos.

Reply:
We do have all the functions in `std::time::nanos` e.g. `from_seconds`